src/Components/Buttons.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import * as api from '../api';
+
+const Buttons = ({votes, showError, updateVotes, loggedInUser, author, id, setComments, setShowDeletedMsg, setShowWrongUserMsg, setShowDeleteError}) => {
+  
+const [notLoggedInError, setNotLoggedInError] = useState(false);
+const [disableDelete, setDisableDelete] = useState(false);
+const [isLoading, setIsLoading] = useState(false);
+
+const handleClick = () => 
+    !loggedInUser ? () => setNotLoggedInError(true) : updateVotes;
+
+const handleDelete = () => {
```

Comment:
I think that we shouldn't be able to see a delete button for comments that aren't the logged in user, not just an error message